Test-NetStack.psm1

```diff
@@ -156,7 +156,14 @@ Function Test-NetStack {
         [Parameter(Mandatory = $false, ParameterSetName = 'OnlyPrereqIPTarget'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesNodes'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesIPTarget', position = 4)]
+
         [Switch] $Experimental = $false,
+        [Parameter(Mandatory = $false)]
+        [String] $DpdkUser='',
```

Comment:
You need to add parameter sets and ensure these params only show up in the correct set.

Otherwise it's confusing to customers how they will use/run the cmdlets (not in stage 8, but these will appear in other places I think).

Reply:
Does it make the most sense to create a new parameter set named "Experimental" and only include these params there?